cw/util/device.py

```diff
@@ -76,8 +76,9 @@ def initialize_fpga(self, fpga, bitstream, pll_frequency):
         # bitstream.
         # Unfortunately, this doesn't seem to work for the CW310 yet,
         # see https://github.com/lowRISC/ot-sca/issues/48.
+        # This should work now. Issue is closed
```

Comment:
I think if the issues is resolved we should actually remove this reference here. :-)